docs/management/connectors/action-types/email.asciidoc

```diff
@@ -5,7 +5,7 @@
 <titleabbrev>Email</titleabbrev>
 ++++
 
-The email connector uses the SMTP protocol to send mail messages, using an integration of https://nodemailer.com/[Nodemailer]. Email message text is sent as both plain text and html text.
+The email connector uses the SMTP protocol to send mail messages, using an integration of https://nodemailer.com/[Nodemailer], except Microsoft Exchange, which use its own integration using https://docs.microsoft.com/en-us/graph/api/overview[Graph API] method https://docs.microsoft.com/en-us/graph/api/user-sendmail[Send mail] on the top of HTTP protocol. Email message text is sent as both plain text and html text.
```

Comment:
Can you break this into two sentences:

The email connector uses the SMTP protocol to send mail messages, using an integration of Nodemailer.  An exception is Microsoft Exchange, which uses the Send mail method in the Graph API on the top of HTTP protocol.

Not sure what this means:

using Graph API method Send mail on the top of HTTP protocol